docs/running-node.md

```diff
@@ -35,21 +35,99 @@ Recommended (for highload applications):
 
 ## Deployment
 
-Pre-requisites:
+### Preparation
 
-1.  `dcld` and `dclcli` binaries are located in `/usr/bin`
-2.  (for systemd systems) `dcld.service` is available in the current directory
-3.  (for validator and observer) `genesis.json` and `persistent_peers.txt` are available in the current directory
+#### (Optional) System cleanup
+
+Required if a host has been already used in another DCLedger setup.
+
+<details>
+<summary>Cleanup (click to expand)</summary>
+<p>
+
+```bash
+$ sudo systemctl stop dcld 
+$ rm -rf "$HOME/.dcld" "$HOME/.dclcli"
+$ sudo rm -f "$(which dcld)" "$(which dclcli)"
+```
+
+</p>
+</details>
+
+#### Get the artifacts
+
+*   download `dclcli`, `dcld` and `dcld.service` from GitHub [release page](https://github.com/zigbee-alliance/distributed-compliance-ledger/releases)
+*   Get setup scripts either from [release page](https://github.com/zigbee-alliance/distributed-compliance-ledger/releases) or
+    from [repository](../deployment/script) if you need latest development version.
+*   (for validator and observer) Get the running DCLedegr network data:
+    *   `genesis.json` can be found in a `<chain-id>` sub-directory of the [persistent_chains](../deployment/persistent_chains) folder and `persistent_peers.txt`
+    *   `persistent_peers.txt`: please contact DCLedger network administrators
+
+<details>
+<summary>Example (click to expand)</summary>
+<p>
+
+```bash
+# release artifacts
+curl -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/<release>/dclcli
+curl -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/<release>/dcld
+curl -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/<release>/dcld.service
+curl -O https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/<release>/run_dcl_node
+
+# OR latest dev version in case of some not yet released improvements
+curl -O https://raw.githubusercontent.com/zigbee-alliance/distributed-compliance-ledger/master/deployment/scripts/run_dcl_node
+
+curl -O https://raw.githubusercontent.com/zigbee-alliance/distributed-compliance-ledger/master/deployment/persistent_chains/<chain-id>/genesis.json
+```
+
+</p>
+</details>
+
+#### Setup DCL binaries
+
+*   located `dlcd` and `dclcli` binaries in a folder listed in `$PATH` (e.g. `/usr/bin/`)
+*   set a proper owner and executable permissions
+
+<details>
+<summary>Example for ubuntu user (click to expand)</summary>
+<p>
+
+```bash
+$ sudo cp ./dclcli ./dcld -t /usr/bin
+$ sudo chmod u+x /usr/bin/dclcli /usr/bin/dcld
+$ sudo chown ubuntu /usr/bin/dclcli /usr/bin/dcld
+```
+
+</p>
+</details>
+
+#### Configure the firewall
+
+Ports `26656` (p2p) and `26657` (RPC) should be available for connections.
+
+<details>
+<summary>Example for Ubuntu (click to expand)</summary>
+<p>
+
+```bash
+$ sudo ufw allow 26656/tcp
+$ sudo ufw allow 26657/tcp
+```
+
+</p>
+</details>
 
 ### Genesis Validator Node
```

Comment:
Let's mention, that usually that step needs to be done just once by a selected Company/participants, so most readers can jump to the Validator Node section.

Reply:
sure, done